src/components/Options/StudentDetailsOptions.tsx

```diff
@@ -4,14 +4,14 @@ export interface OptionType {
 }
 
```

Comment:
Is there some other way to store this options rather than in code? Batches seem quite a lot, and they could keep changing often.

Reply:
Where should we store the options then? maybe a edit button which enables the particular team with access to update the batches